guide/additional-info/miscellaneous-examples.md

```diff
@@ -83,6 +83,7 @@ message.channel.send({ embed: exampleEmbed });
 * There are also specific strings you can use with `.setColor()`, e.g. `DARK_RED` or `RANDOM`. A list of them can be found [here](https://discord.js.org/#/docs/main/stable/typedef/ColorResolvable).
 * `.addBlankField()` is just shorthand for `.addField('\u200b', '\u200b')`. If you want to make it inline as well, pass in `true` as the first argument.
 * If you're on the master branch/v12, you'll need to use `Discord.MessageEmbed()`, not `Discord.RichEmbed()`.
+* If your RichEmbed object is named "embed", you can replace `.send({embed: exampleEmbed});` with `.send({embed})`.
```

Comment:
I think this would be better placed as a `<p class="tip">` tag under "General embed notes" list, since this isn't specific to only the RichEmbed class, but is instead just a JavaScript feature.

In addition:
* 'If your RichEmbed object is named "embed", ...' -> "If your embed object variable is named `embed`, ..."
* Remove the semicolon in the snippet.
* Space out the object key from the curly braces, like the last line in the RichEmbed snippet.

Reply:
Agreed, but `<p class="tip">` won't allow code blocks

Reply:
Maybe not multi-line codeblocks, but inline codeblocks work fine, considering we use them in one of the guide pages already. :P 
![codeblock in a p.tip tag](https://i.imgur.com/FYoT2Dd.png)